src/dbus_fast/introspection.py

```diff
@@ -245,8 +242,10 @@ def __init__(
         name: str,
         signature: str,
         access: PropertyAccess = PropertyAccess.READWRITE,
+        validate: bool = True,
     ):
-        assert_member_name_valid(name)
+        if validate:
```

Comment:
Realized it after merging, but we should probably update the docstrings here as well